.github/workflows/reusable_data_model_gen.yml

```diff
@@ -0,0 +1,67 @@
+name: Reusable Data Model Generator
+
+on:
+  workflow_dispatch:
```

Comment:
For now, make this pipeline to be triggered manually, want to test how it works first